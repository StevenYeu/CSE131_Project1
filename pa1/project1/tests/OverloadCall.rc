function : float foo(int & x) { return 1;}
function : void foo(int x) {}
//function : float foo(float x, float y, float z) { return (x+ y+ z);}

function : int main() {
    foo(1);
    return 0;
}
