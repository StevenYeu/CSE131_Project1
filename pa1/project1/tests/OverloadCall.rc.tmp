Error, "project1/tests/OverloadCall.rc": 
  Duplicate declaration of overloaded function foo.
Error, "project1/tests/OverloadCall.rc": 
  Argument passed to reference parameter x (type int) is not a modifiable L-value.
Compile: failure.
