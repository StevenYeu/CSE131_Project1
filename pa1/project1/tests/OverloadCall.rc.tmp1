  Duplicate declaration of overloaded function foo.
  Argument passed to reference parameter x (type int) is not a modifiable L-value.
Compile: failure.
