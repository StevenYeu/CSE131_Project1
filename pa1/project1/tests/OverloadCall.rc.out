Error, "/tmp/cs131_testrunner_GJ0qMF/file.rc": 
  Duplicate declaration of overloaded function foo.
Error, "/tmp/cs131_testrunner_GJ0qMF/file.rc": 
  Argument passed to reference parameter x (type int) is not a modifiable L-value.
Compile: failure.
